recipes/meg/LICENSE

```diff
@@ -0,0 +1,21 @@
+MIT License
```

Comment:
Please make sure the LICENSE is included in next release

Reply:
LICENSE has been added in source repo